test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -61,10 +61,30 @@ def create_mom_hook(self):
                 'alarm': '3', 'enabled': 'true'}
         self.server.create_import_hook(name, attr, body)
 
+    def create_bad_begin_hook(self):
+        name = "h2"
+        body = ("import pbs\n"
+                "e=pbs.event()\n"
+                "if e.job.in_ms_mom():\n"
+                "    e.accept()\n"
+                "x\n")
+        attr = {'event': 'execjob_begin', 'fail_action': 'offline_vnodes'}
+        self.server.create_import_hook(name, attr, body)
+
+    def create_bad_startup_hook(self):
+        name = "h3"
+        body = ("import pbs\n"
+                "x\n")

```

Comment:
Instead of using 'x' which will cause an exception, why not explicitly raise one?  Right now when someone reads these hooks, they might be confused why it isn't valid python.  If the hook explicitly raises an exception, it will be obvious that you are trying to cause an exception.